### ISSUE TITLE
Which dataset is used for the released pretrained model?

### ISSUE THREAD

**Issue by AArchLichKing** [pengzhiliang/MAE-pytorch#83]
Hey, 

Thanks very much for this excellent repo, it is definitely worth hundreds of thousands of stars! :) 

I wonder which datasets are used for the released pretrained model? Since it works too well to be true for image recovery from randomly masked inputs from FFHQ datasets, especially the backgrounds which has no clues at all for the input part. So is it the case that the model actually saw the data somewhere? 

Thank you very much in advance!!
